jsonld/ldutils_test.go

```diff
@@ -146,9 +149,39 @@ func TestNewContextLoader(t *testing.T) {
 	t.Run("it resolves an external doc when allowingExternalCalls is true", func(t *testing.T) {
 		loader, err := NewContextLoader(true, DefaultContextConfig())
 		assert.NoError(t, err)
-		doc, err := loader.LoadDocument("http://schema.org")
+
+		// Arrange the stubSrv; stubSrv logic is included inline,
+		// it makes the test story telling more clear.
+		//
+		// Stub behavior is based on the code behavior when processing:
+		// loader.LoadDocument("http://schema.org")
+		mux := http.NewServeMux()
+
+		// Stage 1 of the stub
+		f1 := func(w http.ResponseWriter, r *http.Request) {
```

Comment:
`f1` and `f2` could use a more descriptive name, something like `rootHandler` and `schemaHandler` (since they're handler functions)

Reply:
@reinkrul removed the usage of those variables all together.